### PR TITLE
Handle empty includes

### PR DESCRIPTION
**Pull request by thofma**

### CODE REVIEW

**Comment by thofma** [comment]
See https://github.com/Nemocas/AbstractAlgebra.jl/runs/2120118054?check_suite_focus=true.

Should I add a test?

**Comment by benlorenz** [comment]
Thanks, at some point we should probably replace some of the workflows by a runtests.jl that just compares the json output instead of always running the oscar testsuite.

Could you please also bump the version in the Project.toml so we can quickly create a new release once this is merged?

**Comment by thofma** [comment]
Will do once I fixed the error.

File: .github/workflows/oscar-noinclude.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    - name: fetch OscarDevTools
+      if: github.repository != 'oscar-system/OscarDevTools.jl'
+      run: |
+        julia --project=oscar-dev -e "using Pkg;
+                                      Pkg.add(\"OscarDevTools\");
+                                      Pkg.instantiate();"
+    - id: set-matrix
+      run: |
+        julia --project=oscar-dev -e "using OscarDevTools.OscarCI;
+                 ciprefs = parse_meta(\"OscarCI-noinclude.toml\");
```
Comment (benlorenz) [inline .github/workflows/oscar-noinclude.yml:36]:
```suggestion
                 ciprefs = parse_meta(\"test/meta/OscarCI-noinclude.toml\");
```
this needs the full path to the toml, only the default `OscarCI.toml` has a symlink in the top directory.

Comment (thofma) [reply .github/workflows/oscar-noinclude.yml]: Thanks, could have taken me a few minutes figuring this out.

**Review by benlorenz — APPROVED** [review]

**Pull request merged**